python/lsst/sims/photUtils/Photometry.py

```diff
@@ -569,7 +569,8 @@ def sum_magnitudes(self, disk = None, bulge = None, agn = None):
             raise RuntimeError("Arguments of sum_magnitudes need to be " +
```

Comment:
I didn't know where else to put this, but should that say "same type" above on lines 553 & 562?